client/question/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -207,13 +207,20 @@ describe('FeedbackGeneratorService', function() {
           FeedbackGeneratorService._getCorrectnessTestFeedback(
           sampleInputTestCase, sampleInputTestSuite.id, 0,
           'incorrect answer').getParagraphs();
-        expect(CORRECTNESS_FEEDBACK_TEXT.OUTPUT_ENABLED).toContain(
-          correctnessFeedbackParagraphs[0].getContent());
+        expect(correctnessFeedbackParagraphs.length).toEqual(2);
         expect(correctnessFeedbackParagraphs[0].isTextParagraph()).toEqual(
           true);
+        expect(CORRECTNESS_FEEDBACK_TEXT.OUTPUT_ENABLED).toContain(
+          correctnessFeedbackParagraphs[0].getContent());
         expect(correctnessFeedbackParagraphs[1].isOutputParagraph()).toEqual(
           true);
-        expect(correctnessFeedbackParagraphs.length).toEqual(2);
+        var expectedOutputParagraph =
+          'Input: "' + sampleInputTestSuite.testCase.input +
```

Comment:
Hardcode this fully, don't interpolate. This test is supposed to fail when changes to the code result in the output string not matching this one; we should have a "golden string" to compare against.